### ISSUE TITLE
Published package different from dist in local

### ISSUE THREAD

**Issue by jf-niji** [soultice/vite-plugin-vue-hoist-ce-styles#4]
I added the possibility to customize indexRe, but I have an issue when I get the package on the npm repository.

I checked the dist from my local environment and here are the definition of the plugin and the .d.ts files :

**Plugin definition:**

`hoistCeStyles({ hostComponent, indexRe = defaultIndexRe })`

**.d.ts definition:**

```
import type { Plugin } from 'vite';
export declare function hoistCeStyles({ hostComponent, indexRe }: {
    hostComponent: string;
    indexRe: RegExp;
}): Plugin;
```

But on npm package:

**Plugin definition:**

`hoistCeStyles({ hostComponent })`

**.d.ts definition:**

```
import type { Plugin } from 'vite';
export declare const virtualId = "virtual:hoist-ce-styles/css-helper";
export declare function hoistCeStyles({ hostComponent }: {
    hostComponent: string;
}): Plugin;
```

**Comment by soultice** [comment]
Hi, @jf-niji I'm not sure what's going on but I will have a look into it tomorrow. Thanks for letting me know :)

**Comment by soultice** [comment]
shoult be fixed in `0.1.3` messed up the publish process :see_no_evil:

**Comment by soultice** [comment]
closing as should be fixed by now
